cmd/main_test.go

```diff
@@ -54,6 +54,7 @@ type cliTest struct {
 	expectedOutputSubstring string   // output of successful run
 	expectedOutFile         []string // generated output files of successful run
 	expectedErr             []string // expectedErr if assigned, should be returned (at least one of the given options)
+	ensureInputFileExist    string
```

Comment:
it may be sufficient to use `possibleErr` instead, with some sub-string of a file-not-found-err .. this way the test will be skipped and not failed.

Reply:
fixed